### ISSUE TITLE
Edit playlist page has many accessibility violations

### ISSUE THREAD

**Issue by cjcolvar** [avalonmediasystem/avalon#3081]
Using wave on the edit playlist page on MCO shows 43 errors mostly focused on form inputs missing labels or incorrectly associated labels.

**Comment by joncameron** [comment]
Already handled in other issues.
